src/multiselectMixin.js

```diff
@@ -2,7 +2,7 @@ import deepClone from './utils'
 
 function isEmpty (opt) {
   if (opt === 0) return false
-  if (opt === []) return true
```

Comment:
`opt === []` always equals false, since you are comparing references rather than values.